packages/syft/setup.cfg

```diff
@@ -61,6 +61,7 @@ syft =
     tqdm==4.64.0
     typeguard==2.13.3
     typing_extensions==4.4.0
+    ipython==8.5.0
```

Comment:
Do we need to make this mandatory with the syft package?

Reply:
I tried to eliminate it and I had an issue with running kernels in vscode. We can eliminate it and keep in mind if it comes up as an issue for the users